packages/block-editor/src/components/text-decoration-control/README.md

```diff
@@ -0,0 +1,37 @@
+# TextDecorationControl
+
+**Notice: This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.**
```

Comment:
```suggestion
<div class="callout callout-alert">
This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
</div>
<br />
```

I believe these readme files can be pulled into other documentation such as the Gutenberg Handbook. In those cases, using the `div` with callout classes helps highlight this important note.

You can see this already in use for the [Block Variation Picker component](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-variation-picker/README.md?plain=1#L3C1-L5).